.eslintrc.json

```diff
@@ -26,23 +26,50 @@
     "no-array-constructor": "error",
     "no-bitwise": "error",
     "no-caller": "error",
+    "no-catch-shadow": "error",
```

Comment:
+1.

This is IE<8 specific, so seems redundant. However, given that general block-level variables don't exist in ES5 (only ES6) this one occurrence of block-level variables is probably confusing if we allow it, so best to keep socially enforced that variable names are unique within the entire function.